packages/dm-core-plugins/src/form/templates/shared/PrimitiveArray.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button, Icon, Tooltip } from '@equinor/eds-core-react'
+import { add } from '@equinor/eds-icons'
+import React, { useState } from 'react'
+import { AttributeField } from '../../fields/AttributeField'
+import { TUiAttributeObject } from '../../types'
+import { DeleteSoftButton, TAttribute } from '@development-framework/dm-core'
+
+interface PrimitiveArrayProps {
```

Comment:
I'm guessing the `any` usages in this interface could be replaced by something more typesafe?

Reply:
Done